packages/build/src/github-repo.ts

```diff
@@ -208,6 +209,27 @@ export class GithubRepo {
     return `https://jira.mongodb.org/issues/?jql=project%20%3D%20MONGOSH%20AND%20fixVersion%20%3D%20${version}`;
   }
 
+  async createPullRequest(title: string, fromBranch: string, toBaseBranch: string): Promise<{prNumber: number, url: string}> {
+    const response = await this.octokit.pulls.create({
+      ...this.repo,
+      base: toBaseBranch,
+      head: fromBranch,
+      title
+    });
+
+    return {
+      prNumber: response.data.number,
+      url: response.data.html_url
+    };
+  }
+
+  async mergePullRequest(prNumber: number): Promise<void> {
+    await this.octokit.pulls.merge({
+      ...this.repo,
+      pull_number: prNumber
+    });
+  }
```

Comment:
Does whichever github user is running in CI have merge permissions for the repo?

Reply:
That's a great question, i don't think so, and i actually have no clue who we should ask, going to ask Jess, probably she knows